pkgs/development/python-modules/torchmetrics/default.nix

```diff
@@ -40,9 +42,9 @@ buildPythonPackage {
   ];
 
   # Let the user bring their own instance
-  buildInputs = [
-    torch
-  ];
+  buildInputs = [ torch ];
+  # A cyclic dependency in: integrations/test_lightning.py
+  doCheck = false;
```

Comment:
Also no need to move this around, I think. Actually, having `doCheck = false` near `passthru.tests.check` is even convenient